app/views/take_whole_pot_calculator/show.html.erb

```diff
@@ -0,0 +1,40 @@
+<h1>Taking your whole pot in one go</h1>
+
+<div class="calculator">
+  <h2>Estimate how much you would get</h2>
+
+  <form action="/take-whole-pot/results" method="post">
```

Comment:
My original thought was that we were "creating" a calculation, but what you say has merit: we're not creating anything.